clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/api/KubernetesClientApiConverter.groovy

```diff
@@ -964,7 +964,11 @@ class KubernetesClientApiConverter {
     return daemonset
   }
 
-  static boolean canUpdated(DeployKubernetesAtomicOperationDescription description) {
+  static Boolean canUpdated(DeployKubernetesAtomicOperationDescription description) {
```

Comment:
Can you name this more clearly? "canUpdated" isn't clear on what the intent is here.

Reply:
To ensure the updateController field is presented.